frontend/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_SOLUTION_VERSION=
```

Comment:
Is it a mandatory file?  Could we only set the environment variable `REACT_APP_SOLUTION_VERSION` when building frontend code?